<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="6" passed="5" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-02-13T16:26:13 IST" name="Suite" finished-at="2024-02-13T16:27:14 IST" duration-ms="61317">
    <groups>
    </groups>
    <test started-at="2024-02-13T16:26:13 IST" name="Test" finished-at="2024-02-13T16:27:14 IST" duration-ms="61317">
      <class name="test.main">
        <test-method is-config="true" signature="driverSetup(java.lang.String)[pri:0, instance:test.main@3c77d488]" started-at="2024-02-13T16:26:13 IST" name="driverSetup" finished-at="2024-02-13T16:26:28 IST" duration-ms="14818" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- driverSetup -->
        <test-method signature="check()[pri:0, instance:test.main@3c77d488]" started-at="2024-02-13T16:26:28 IST" name="check" finished-at="2024-02-13T16:26:29 IST" duration-ms="876" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- check -->
        <test-method signature="count_news()[pri:1, instance:test.main@3c77d488]" started-at="2024-02-13T16:26:29 IST" name="count_news" finished-at="2024-02-13T16:26:29 IST" duration-ms="50" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- count_news -->
        <test-method signature="check_tt()[pri:2, instance:test.main@3c77d488]" started-at="2024-02-13T16:26:30 IST" name="check_tt" finished-at="2024-02-13T16:26:30 IST" duration-ms="309" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- check_tt -->
        <test-method signature="i_o_header()[pri:3, instance:test.main@3c77d488]" started-at="2024-02-13T16:26:30 IST" name="i_o_header" finished-at="2024-02-13T16:26:53 IST" duration-ms="23138" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [CEO blog: Elevating Cognizant’s stature on the world stage] but found [Our growth mindset]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [CEO blog: Elevating Cognizant’s stature on the world stage] but found [Our growth mindset]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at pages.home.i_o_headers(home.java:117)
at test.main.i_o_header(main.java:48)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- i_o_header -->
        <test-method signature="news_Execs()[pri:4, instance:test.main@3c77d488]" started-at="2024-02-13T16:26:54 IST" name="news_Execs" finished-at="2024-02-13T16:27:10 IST" duration-ms="16503" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- news_Execs -->
        <test-method signature="apps_tools()[pri:5, instance:test.main@3c77d488]" started-at="2024-02-13T16:27:10 IST" name="apps_tools" finished-at="2024-02-13T16:27:13 IST" duration-ms="2389" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- apps_tools -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:test.main@3c77d488]" started-at="2024-02-13T16:27:13 IST" name="closeBrowser" finished-at="2024-02-13T16:27:14 IST" duration-ms="1164" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- test.main -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
